client/src/ui/modules/military/battle-view/BattleActions.tsx

```
@@ -180,38 +190,58 @@
   );
 
   const mouseEnterRaid = useCallback(() => {
+    const troopConfig = getComponentValue(TroopConfig, getEntityIdFromKeys([WORLD_CONFIG_ID]));
+    if (!troopConfig) return 0;
+
+    const raidSuccessPercentage = getChancesOfSuccess(selectedArmy, defenderArmy, troopConfig) * 100;
+
+    const maxResourceAmountStolen = getMaxResourceAmountStolen(selectedArmy, defenderArmy, troopConfig);
+    const [attackerTroopsLoss, defenseTroopsLoss] = getTroopLossOnRaid(selectedArmy, defenderArmy, troopConfig);
+    let content = [
+      <div>Raid outcome:</div>,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)